google_photos_library_api/exceptions.py

```diff
@@ -15,7 +15,3 @@ class AuthException(GooglePhotosApiError):
 
 class ApiForbiddenException(GooglePhotosApiError):
     """Raised due to permission errors talking to API."""
-
-
-class GooglePhotosParseException(GooglePhotosApiError):
```

Comment:
The reason this is here is to avoid exposing the underlying parsing format e.g. how [gcal_sync](https://github.com/allenporter/gcal_sync/blob/511563d102036ee9bbdb8a204ee855042b190d8e/gcal_sync/model.py#L111) handles this.  The idea is we don't want expose to upstream libraries that this depends on mashumaro. This came up via feedback from home assistant to avoid having tests depend on that level of detail. 

I'm happy to drop this for now and i can add it back when i fix it here, but you may want to improve this in your library also.